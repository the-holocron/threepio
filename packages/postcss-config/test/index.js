const config = require("../postcss.config.js");

describe("configuration", () => {
	describe("postCSS", () => {
		test("should render the configuration", () => expect(config).toMatchSnapshot());
	});
});
