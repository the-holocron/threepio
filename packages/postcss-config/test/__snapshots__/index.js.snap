// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configuration PostCSS should render the configuration 1`] = `
Object {
  "plugins": Array [
    [Function],
    [Function],
  ],
}
`;
